### ISSUE TITLE
Fell landing Poisonous Brew Cauldron mechanic

### ISSUE THREAD

**Issue by BomblingTV** [WarEmu/WarBugs#14720]
Hello,

The Poison from the Fell landing (and perhaps Reikwald) Fortress mechanic seem to have a few errors depending on what the intent is.
First of all the Brew seem to be unable to get attacked by melee hits:
https://clips.twitch.tv/InspiringObedientPorcupineSMOrc

And from recording the last 3 Fell Landing attacks, groundtarget AoE like Rain of Fire, Napalm Grenade and even Blast potions dont damage the brew either
- That could be a result of the Brew respawning too fast when the Fortress lord is resetting constantly (as a result of a well functioning defensive funnel or bodyblock/tankwall)
https://www.twitch.tv/videos/462153052

On top of that, the damage is unmitigated moral-damage and tank M4s are not helping the attackers mitigate the enormous presure, so as soon as the attacking realm step into player-damage on top of the moral brew, they just explode
https://www.twitch.tv/videos/462015966

I made 1 VOD review with commentary over attacking point of view:
https://www.twitch.tv/videos/462015966
1 defenders point of view, just looking at how the brew and lord reacts to the attackers:
https://www.twitch.tv/videos/462153052
and an additional defenders point of view to see if the brew could even be hit and killed from ranged:
https://www.twitch.tv/videos/462343089

Having one set of fortress mechanics being moral-damage based compared to "only elves are able to build aggro" is such a massive difference in difficulty and probably should be adjusted.

**Comment by Natherul** [comment]
Ill just add that difficulty of different forts is INTENDED and that the aggro thing was set wrong and has been addressed.


Idea is that once the realms know the mechanics (and we will not help with saying what they are) the realm will decide to push the fort that are best for them unless they have no choice because others are locked.  And even so attacking a fort should be faar from a win. A fort SHOULD be hard to take.

**Comment by BomblingTV** [comment]
I understand and respect that forts need to be hard, and not an any feed. But this mechanic still seem to be overtuned, by a long shot. Funnels are hard enough to break as if, 

Right now if you try and break the funnel, the cauldron will break you. 
If you try and break the cauldron, the players and/or lord will break you.
If you try and control the lord, the funnel and/or cauldron will break you.

its currently a spiral where all of the bases cover eachother. Been assigning people to running all over the fort zones and look during stage 1,2 and 3 for any antidote cauldron the attackers need to zip or whatever counterplay mechanic should be in play.
If the counter play is simply to controle the lord and then killing the cauldron, thats simply not enough of a counter imho. Speaking from both Destro and Order point of view, and with the mindset of I dont want forts to be easy.

I just tested Reikwald in the Early EU time, after a hotfix about 10hours ago where The respawn rate and damage should had been reduced by 30%. This didnt seem to be the case in Reikwald where everyone on the attacking side still took 1099 moral unmitigateable damage.
https://www.twitch.tv/videos/462745403?t=00h15m35s

The cauldron also constantly respawns, on lord reset. Is there anyway it will just stay on the ground after lord has been aggroed, it needs to be killed before it auto despawns 30sec later, with an internal 30sec cooldown so it can spawn again on next lordaggro. Therefor giving the attackers a kill window on first aggro when they choose to aggro lord first time, kill the cauldron and deal with the lord-aggro and players in a 30sec window before next cauldron can spawn. 

in the end of the VOD i linked, we found a way to bug out the lord aggro that only worked in reikwald layout, so a healer can keep lord aggro out of cauldron range, keeping lord aggroed and controled on threat, canceling out the mechanic, and i wouldnt even be supriced if this is the only reason this mechanic has been rumored to being overcome once by destro and not by order (tried this method on orderside too)
